src/main/java/org/commcare/cases/query/QueryCache.java

```diff
@@ -0,0 +1,47 @@
+package org.commcare.cases.query;
+
+import java.util.HashMap;
+
+/**
+ * Created by ctsims on 1/26/2017.
+ */
+
+public class QueryCache {
```

Comment:
Updated the docs to reflect this a bit better(push is coming with the updated docs)